k8s_adapter/mcp/terraform/kubernetes.tf

```diff
@@ -0,0 +1,769 @@
+                }
+              }
+            }
+            dynamic "readiness_probe" {
+              for_each = lookup(container.value, "readiness_probe", null) == null ? {} : { readiness_probe : container.value.readiness_probe }
+              content {
+                initial_delay_seconds = lookup(readiness_probe.value, "initial_delay_seconds", null)
+                period_seconds        = lookup(readiness_probe.value, "period_seconds", null)
+                timeout_seconds       = lookup(readiness_probe.value, "timeout_seconds", null)
+                success_threshold     = lookup(readiness_probe.value, "success_threshold", null)
+                failure_threshold     = lookup(readiness_probe.value, "failure_threshold", null)
+                dynamic "http_get" {
+                  for_each = lookup(readiness_probe.value, "http_get", null) == null ? {} : { http_get : readiness_probe.value.http_get }
+                  content {
+                    host   = lookup(http_get.value, "host", null)
+                    path   = lookup(http_get.value, "path", null)
+                    scheme = lookup(http_get.value, "scheme", null)
+                    port   = lookup(http_get.value, "port", null)
+                    dynamic "http_header" {
+                      for_each = lookup(http_get.value, "http_header", null) == null ? {} : { http_header : http_get.value.http_header }
+                      content {
+                        name  = http_header.value.name
+                        value = http_header.value.value
+                      }
+                    }
+                  }
+                }
+                dynamic "tcp_socket" {
+                  for_each = lookup(readiness_probe.value, "tcp_socket", null) == null ? {} : { tcp_socket : readiness_probe.value.tcp_socket }
+                  content {
+                    port = tcp_socket.value.port
+                  }
+                }
+                dynamic "exec" {
+                  for_each = lookup(readiness_probe.value, "exec", null) == null ? {} : { exec : readiness_probe.value.exec }
+                  content {
+                    command = exec.value.command
+                  }
+                }
+              }
+            }
+            dynamic "volume_mount" {
+              for_each = lookup(container.value, "volume_mount", null) == null ? {} : { volume_mount : container.value.volume_mount }
+              content {
+                mount_path        = lookup(volume_mount.value, "mount_path", null)
+                name              = lookup(volume_mount.value, "name", null)
+                read_only         = lookup(volume_mount.value, "read_only", null)
+                sub_path          = lookup(volume_mount.value, "sub_path", null)
+                mount_propagation = lookup(volume_mount.value, "mount_propagation", null)
+
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+resource "kubernetes_service" "self" {
```

Comment:
split service, deployment and other adapters into their own files

Reply:
done